jsonschema_extractor/tests/test_attrs_validators.py

```diff
@@ -0,0 +1,45 @@
+import attr
+from typing import Optional
+
+POSSIBLE_OPTIONS = ['option1', 'option2']
+
+
+@attr.s
+class Example(object):
+    options = attr.ib(type=str, validator=attr.validators.in_(POSSIBLE_OPTIONS))
+
+
+@attr.s
+class ExampleWithoutType(object):
+    options = attr.ib(validator=attr.validators.in_(POSSIBLE_OPTIONS))
+
+
+def test_in_validator_happy_flow(extractor):
```

Comment:
I would add test cases to catch the situations that may have issues above:

1. a test where the attrs.in_ passes in an enum.Enum.
2. passing in a situation where schema is not possible to extract, and there is an in_ validator (should raise unextractable schema, I think it'll raise valueerror now because the key is not set).

Reply:
Added the wanted test cases and a test case for `tuple` and `set` :)